lib/shopify-cli/monorail.rb

```diff
@@ -0,0 +1,109 @@
+require 'json'
+require 'securerandom'
+require 'time'
+require 'timeout'
+require 'net/http'
+
+module ShopifyCli
+  module Monorail
+    autoload :Log, 'shopify-cli/monorail/log'
+
+    ENDPOINT_URI = URI.parse('https://monorail-edge.shopifycloud.com/v1/produce')
+
+    MonorailError = Class.new(StandardError)
```

Comment:
Why `MonorailError = Class.new(StandardError)` over `class MonorailError << StandardError; end`?

Reply:
This was a hamfisted copy from dev, I like the explicit declaration better.